src/emulator/commandUtils.ts

```diff
@@ -5,6 +5,19 @@ import requireAuth = require("../requireAuth");
 import requireConfig = require("../requireConfig");
 import { Emulators } from "../emulator/types";
 
+export class Flags {
+  static FLAG_ONLY: string = "--only <list>";
```

Comment:
this may be out of scope (though it doesn't really change much):

`--only <list>` is... vague. The deploy command says `<targets>` which is pretty obvious. Could this be `<emulators>`? What would a tech writer say?

Reply:
Idea: `<emulator>[,<emulator>...]` like unix man pages?

Reply:
I think that makes Commander choke, though I like the idea.

`<"noun"s>` is consistent with the rest of the CLI, too.

Reply:
Going with `emulators` since that's the best we can do with Commander.